projectm-eval/TreeFunctions.c

```diff
@@ -21,6 +21,8 @@ static prjm_eval_function_def_t intrinsic_function_table[] = {
     { "/*const*/", prjm_eval_func_const,         0, true,  false },
     { "/*var*/",   prjm_eval_func_var,           0, false, false },
     { "/*list*/",  prjm_eval_func_execute_list,  1, true,  false },
+    { "/*or*/",    prjm_eval_func_or,            2, true, false },
+    { "/*and*/",   prjm_eval_func_and,           2, true, false },
```

Comment:
Using the special "comment" syntax here as ns-eel2 doesn't have any internal name for these two operator implementations. Since the scanner will skip over comments, this text can never occur as a function token in the parser and thus doesn't influence the syntax in any way.